lib/image-diff.js

```diff
@@ -54,17 +54,20 @@ ImageDiff.createDiff = function (options, cb) {
   // http://www.imagemagick.org/script/compare.php
   var diffCmd = 'compare';
   var diffArgs = [
-    'compare',
     '-verbose',
     // TODO: metric and highlight could become constructor options
     '-metric', 'RMSE',
-    '-highlight-color', 'RED',
-    '-compose', 'Src',
+    '-dissimilarity-threshold', '1',
+    '-highlight-color', 'RED']
+    //shadow options if options.shadow is set
+    .concat(options.shadow ? ['-compose', 'Src'])
```

Comment:
The test suite is failing due to no `else` in this ternary

https://travis-ci.org/uber/image-diff/jobs/68629732

Can you update it to:

```js
.concat(options.shadow ? ['-compose', 'Src'] : [])
```